fourth review/swagger.yaml

```diff
@@ -0,0 +1,1366 @@
+        readOnly: true
+
+      line_items:
+        type: array
+        description: Product variant line item or custom line item associated to the draft order.  Each draft order must include at least one `line_item`.
+        items:
+          $ref: '#/definitions/line_item'
+
+      metafields:
+        $ref: '#/definitions/metafield'
+
+      shipping_line:
+        $ref: '#/definitions/shipping_line'
+
+      tags:
+        type: string
+        description: Additional short descriptors, commonly used for filtering and searching, formatted as a string of comma-separated values. Each individual tag is limited to 40 characters in length.
+      tax_exempt:
+        type: boolean
+        description: Sets whether or not taxes are exempt for this draft order. If this value is `false`, Shopify will honor the `taxable` setting for each `line_item`.
+        enum:
+        - true
+        - false 
+      tax_lines:
+        type: array
+        description: An array of `tax_line` objects, each of which details the applicable taxes.
+        items:
+          $ref: '#/definitions/tax_line'
+        readOnly: true
+      applied_discount:
+        $ref: '#/definitions/applied_discount'
+      taxes_included:
+        type: boolean
+        description: States whether or not taxes are included in the draft order subtotal.
+        readOnly: true
+        enum:
+        - true
+        - false
+
+      total_tax:
+        type: string
+        description: Total tax amount.
+        readOnly: true
+
+      subtotal_price:
+        type: string
+        description: Subtotal after discounts, but before shipping and taxes have been applied.
+        readOnly: true
+
+      total_price:
+        type: string
+        description: Includes discounts, shipping and taxes.
+        readOnly: true
+
+      completed_at:
+        type: string
+        format: date-time
+        description: Date at which order is created and the draft order changes to “completed” status.
+        readOnly: true
+      created_at:
+        type: string
+        format: date-time
+        description: By default, this auto-generated property is the date and time when the draft order was created in Shopify, in ISO 8601 format. 
+        readOnly: true
+
+      updated_at:
+        type: string
+        format: date-time
+        description: The date and time when the draft order was last updated.
+        readOnly: true
+
+      status:
+        type: string
+        enum:
+        - open
+        - invoice_sent
+        - completed
+        description: String describing the state of the draft order.
+        readOnly: true
+
+  draft_order_invoice:
+    properties:
+      to:
+        type: string
+        description: >-
+          Email where the `draft_order_invoice` will be sent.  If left blank the email associated to the draft order is used instead; otherwise, the customer email is used.  If all three possible email sources are empty, an error is returned.
```

Comment:
* Begin "The email address where..."
* "If left blank, then the email address associated with the draft order is used... etc."